### ISSUE TITLE
Homepage A11y: Heading levels

### ISSUE THREAD

**Issue by josieg** [BCcampus/open-sage#146]
**Describe the bug**
The heading levels on the homepage are used to set style, rather than semantic meaning. For example, right now "Create Open Textbooks" "Use Open Textbooks" and "Open Education at BCcampus" are all set as heading 4, which implies that they fall under the topic of "Search the open textbook collection" which is heading 3.

See: https://helga.bccampus.ca/wp/opentextbooks/

**Expected behavior**
I expect that heading levels should convey semantic meaning - i.e., convey how different topics relate to each other. 

- Missing a heading 1 - if there is no higher heading, then all following headings should be bumped up one heading level from what I have them listed as
- h2: Find Open Textbooks
- h3: Search the B.C. Open Textbook Collection
- h2: Create Open Textbooks
- h2: Use Open Textbooks
- h2: Open Education at BCcampus
- h2: Getting Started

**Screenshots**
![homepageWAVE](https://user-images.githubusercontent.com/23706762/54765461-cdc6ce00-4bb6-11e9-9125-40852d089a1b.jpg)

**Comment by alex-418** [comment]
Completed via https://github.com/BCcampus/open-sage/commit/87d7c72d74301d6712e970220ad28a28a5594a14
